jib-plugins-common/gradle.properties

```diff
@@ -0,0 +1 @@
+version = 0.1.0-SNAPSHOT
```

Comment:
tiny nit: Looks like we don't have ending newlines for the gradle.properties but probably should just add them